thirdparty/maven/zimbra-maven/debian/watch

```diff
@@ -0,0 +1,2 @@
+version=3
+http://archive.apache.org/dist/maven/maven-3/3.3.9/source/apache-maven-(\d+\.\d+\.\d+).tar.xz
```

Comment:
Agreed, we use variables for substitution URLs that have the version in it in other Makefile's as well.